packages/html/src/upload/tests/upload-states.tsx

```diff
@@ -0,0 +1,29 @@
+import { Upload, UploadChunk } from '../../upload';
```

Comment:
In my opinion, here we could use UploadNormal, as disabled Upload does not allow to upload files